provider/lxd/environ_raw.go

```diff
@@ -6,20 +6,46 @@
 package lxd
 
 import (
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
 	"github.com/juju/errors"
+	"github.com/juju/utils"
+	lxdshared "github.com/lxc/lxd/shared"
 
+	"github.com/juju/juju/environs"
+	jujupaths "github.com/juju/juju/juju/paths"
 	"github.com/juju/juju/network"
 	"github.com/juju/juju/provider/common"
 	"github.com/juju/juju/tools/lxdclient"
 )
 
+var (
+	jujuConfDir    = jujupaths.MustSucceed(jujupaths.ConfDir("xenial"))
```

Comment:
paths.ConfDir expects a series. We need an Ubuntu series. Do you have an alternative suggestion?